assets/js/search-data.json

```diff
@@ -0,0 +1,12 @@
+[
+  {{ range $index, $page := .Site.Pages }}
+    {{ if ne $index 0 }},{{ end }}
```

Comment:
If there is a better way of adding the comma between objects in the array (some variant of map and join?) I would be happy to change this.

Reply:
Sadly not. We already use a similar construct in another theme already https://github.com/thegeeklab/hugo-geekblog/blob/master/layouts/_default/list.json.json#L54 so that's totally fine :)